### PR TITLE
114 added computed (tech debt)

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/src/views/PDFSigner.jsx [hunk 1]
```diff
@@ -39,7 +35,7 @@ class PDFSignerComponent extends React.Component {
 
     this.props.pdfObj.getPage(pageNumber).then(page => {
       const scale = 1;
-      const viewport = page.getViewport(scale);
+      const viewport = page.getViewport({ scale });
```
Comment (sutt0n) [inline web-client/src/views/PDFSigner.jsx:38]: This gets rid of a deprecated error being output in the console

**Review by codyseibert — APPROVED** [review]

File: web-client/src/views/PDFSigner.jsx [hunk 2]
```diff
@@ -57,12 +53,17 @@ class PDFSignerComponent extends React.Component {
...
+    this.props.setSignatureData({
+      signatureApplied: false,
+      signatureData: null,
+    });
   }
 
   stop(canvasEl, sigEl, x, y, scale = 1) {
-    this.props.setSignatureData({ signatureData: { scale, x, y } });
+    this.props.setSignatureData({
+      signatureApplied: true,
```
Comment (matthopson) [inline web-client/src/views/PDFSigner.jsx:64]: What's this for?

File: web-client/src/views/PDFSigner.jsx [hunk 3]
```diff
@@ -75,9 +76,10 @@ class PDFSignerComponent extends React.Component {
     let x;
     let y;
 
-    this.setState({ signatureApplied: true });
-
-    this.props.setSignatureData({ signatureData: null });
+    this.props.setSignatureData({
+      signatureApplied: true,
```
Comment (matthopson) [inline web-client/src/views/PDFSigner.jsx:80]: Should this be false?

File: web-client/src/views/PDFSigner.jsx (hunk 2 above)
Comment (matthopson) [reply web-client/src/views/PDFSigner.jsx:64]: Ah, I see - before we were checking if it was null.

**Pull request merged**